### DISCUSSION TITLE
what is the numbers in upper left corner?

### DISCUSSION THREAD

**Discussion by DariBer** [Blackymas/NSPanel_HA_Blueprint#294] [Q&A] [answered]
I wonder if someone could explain what the status icon up in the left corner is, the one with 1 or 2 in a square.

I can’t find out what they are for.

**Comment by deejaybeam** [comment, marked as answer]
This are the states of the relays

**Reply by DariBer** [reply]
ahhh!!!! 

That's it.. :-)
